src/components/nc-slideshow.vue

```diff
@@ -0,0 +1,249 @@
+<template>
+  <div class="nc-slideshow">
+    <div class="nc-slideshow__content" :style="{'width': `${width}px`}">
+      <ul class="list" v-touch:swipe="goToSlide"
+        :style="{
+          width: `${width * slideLength}px`,
+          transform: `translate3d(-${slideIndex * width}px, 0px, 0px)`,
+          transition: '-webkit-transform 500ms ease'
```

Comment:
Esto no va hacer que sólo funcione en chrome?
```suggestion
          transition: 'transform 500ms ease'
```

Reply:
Eliminado